utils/index.js

```diff
@@ -1,4 +1,19 @@
 const { execSync } = require('child_process');
+const packageJson = require(`${process.cwd()}/package.json`);
```

Comment:
read the file inside the function instead with `fs`, this require may be cached and depending on how actions work you may have a cached package.json of a different package